app/controllers/ApiController.scala

```diff
@@ -29,11 +29,17 @@ case class ApiController @Inject() (
 
   private def matchFranceServiceInstance(
       franceServiceInstance: FranceServiceInstance,
-      groups: List[UserGroup]
+      groups: List[UserGroup],
+      doNotMatchTheseEmails: List[String]
```

Comment:
```suggestion
      doNotMatchTheseEmails: Set[String]
```
optimisation gratuite